dev/src/Command/AddComponentCommand.php

```diff
@@ -184,21 +210,215 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $composer->updateMainComposer();
         $composer->createComponentComposer($new->displayName, $new->githubRepo);
 
+        $this->validateOptions($input, $output);
+        if ($input->getOption('bazel-path')) {
+            $googleApisDir = realpath($input->getOption('bazel-path'));
+            $output->writeln("\n\nbazel build library");
+            $output->writeln($this->bazelQueryAndBuildLibrary($new->protoPath, $googleApisDir));
+            $output->writeln("\n\nCopying the library code from bazel-bin");
+            $output->writeln($this->owlbotCopyBazelBin($new->componentName, $googleApisDir));
+            $output->writeln("\n\nOwlbot post processing");
+            $output->writeln($this->owlbotPostProcessor());
+        } else if ($input->getOption('googleapis-gen-path')) {
+            $this->checkDockerAvailable();
+            $googleApisGenDir = realpath($input->getOption('googleapis-gen-path'));
+            $output->writeln("\n\nCopying the library code from googleapis-gen");
+            $output->writeln($this->owlbotCopyCode($new->componentName, $googleApisGenDir));
+            $output->writeln("\n\nOwlbot post processing");
+            $output->writeln($this->owlbotPostProcessor());
+        }
```

Comment:
nit: since we have the last two lines in common for both, maybe it would be better to do something like this? 

```php
if ($input->getOption('bazel-path')) {
    // ...
} elseif ($input->getOption('googleapis-gen-path')) {
    // ...
}

// run owlbot post-processor if a bazel-path or googleapis-gen-path was supplied
if ($input->getOption('bazel-path') || $input->getOption('googleapis-gen-path')) {
    $output->writeln("\n\nOwlbot post processing");
    $output->writeln($this->owlbotPostProcessor());
}
```

If you prefer it the other way, please disregard! This is only a matter of preference